tests/test_auth_api.py

```
@@ -0,0 +1,135 @@
+
+
+def test_guard_rejects_logged_out(client):
+    resp = client.get("/api/git/commits")
+    assert resp.status_code == 401
+    assert resp.get_json()["success"] is False
+
+
+def test_guard_allowlist_stays_public(client):
+    # /api/version 公开
+    assert client.get("/api/version").status_code == 200
+    # /api/auth/me 虽返回 401，但走自身逻辑，并非被守卫拦截（路径在白名单）
```

Ruff (0.15.17)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)